upi/openstack/down-network.yaml

```diff
@@ -4,65 +4,65 @@
 # openstackclient
 # openstacksdk
 
-- import_playbook: common.yaml
+- ansible.builtin.import_playbook: common.yaml
 
 - hosts: all
   gather_facts: no
 
   tasks:
   - name: 'List ports attatched to router'
-    command:
+    ansible.builtin.command:
       cmd: "openstack port list --device-owner=network:router_interface --tags {{ cluster_id_tag }} -f value -c id"
     register: router_ports
 
   - name: 'Remove the ports from router'
-    command:
+    ansible.builtin.command:
       cmd: "openstack router remove port {{ os_router }} {{ item.1}}"
     with_indexed_items: "{{ router_ports.stdout_lines }}"
 
   - name: 'List ha ports attached to router'
-    command:
+    ansible.builtin.command:
       cmd: "openstack port list --device-owner=network:ha_router_replicated_interface --tags {{ cluster_id_tag }} -f value -c id"
     register: ha_router_ports
 
   - name: 'Remove the ha ports from router'
-    command:
+    ansible.builtin.command:
       cmd: "openstack router remove port {{ os_router }} {{ item.1}}"
     with_indexed_items: "{{ ha_router_ports.stdout_lines }}"
 
   - name: 'List ports'
-    command:
+    ansible.builtin.command:
```

Comment:
we could use port collection instead https://docs.ansible.com/ansible/latest/collections/openstack/cloud/port_info_module.html#ansible-collections-openstack-cloud-port-info-module

Reply:
yes, let's revisit that later if possible.